lib/castle/api.rb

```diff
@@ -2,44 +2,32 @@
 
 module Castle
   # this class is responsible for making requests to api
-  class API
-    def initialize(headers = {})
-      @config = Castle.config
-      @http = prepare_http
-      @headers = headers.merge('Content-Type' => 'application/json')
-    end
-
-    def request(command)
-      request = Castle::Request.new(@headers).build(
-        command.path,
-        command.data,
-        command.method
-      )
-      perform_request(request)
-    end
-
-    private
+  module API
+    EXCEPTIONS = [
```

Comment:
we could call it "handled exceptions" as I initially though this is list of our exceptions